pkg/features/kube_features.go

```diff
@@ -485,6 +485,7 @@ const (
 	// owner: @andrewsykim @adisky
 	// alpha: v1.20
 	// beta: v1.24
+	// ga: v1.26
```

Comment:
nit: `GA`

Reply:
Also, feel free to add your name to the owners list on line 485

Reply:
Done! Feels amazing to be an owner of my first feature in OSS :D

Reply:
congrats @ndixita !